auth/http.go

```diff
@@ -1266,6 +1263,22 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 	broadcastGroupChange(central, r, groupname)
 }
 
+func permListToText(central *ImqsCentral, added *authaus.PermissionList) string {
+	s := ""
+	if added == nil {
+		central.Central.Log.Warnf("permListToText: 'added' list is nil")
```

Comment:
Personally, I would want it to be logged as an error but I could also understand that the show went on without it so warning although we return "<error>".